build

```diff
@@ -63,7 +68,12 @@ if [ -n "$build_dep_dir" ]; then
 fi
 
 if [ -z "$skip_source" ]; then
-	podman run --security-opt seccomp=unconfined --arch "$arch" --rm "${mount_opts[@]}" -w "/opt/package_build/workdir" "$container" /opt/package_build/bin/source
+  if [ -z "$gl_package_build_debug" ]; then
+	  podman run --security-opt seccomp=unconfined --arch "$arch" --rm "${mount_opts[@]}" -w "/opt/package_build/workdir" "$container" /opt/package_build/bin/source
+  else
+    echo "Starting package build container without --rm"
+	  podman run --security-opt seccomp=unconfined --arch "$arch" "${mount_opts[@]}" -w "/opt/package_build/workdir" -e gl_package_build_debug=true "$container" /opt/package_build/bin/source
+  fi
```

Comment:
Should the same logic also apply for the binary build? I think chances are higher that there are issues in the source build, but depending on how the package is built issues might also come up in the binary part (I had that in one package so far, not sure if it's common)

Open question, I'm fine the way it is.